### PR TITLE
Added the documentation for the info end point

### PR DESCRIPTION
**Pull request by EOSullivanBerlin**

### CODE REVIEW

File: src/06-products/products.apib [hunk 1]
```diff
@@ -24,3 +24,22 @@ Get a list of products.
...
+
+    + Attributes (object)
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+
+ + Response 201 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+            + name: `cookies` (string)
```
Comment (dcsg) [inline src/06-products/products.apib]: Q: since `name`, `code` and `description` are optional should they be nullable here as well?

Comment (EOSullivanBerlin) [reply src/06-products/products.apib]: Fair enough I will change them.

File: src/06-products/products.apib [hunk 2]
```diff
@@ -24,3 +24,22 @@ Get a list of products.
...
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+
+ + Response 201 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+            + name: `cookies` (string, nullable)
+            + description: `dark chocolate` (string, nullable)
+            + code: `1523999er2323712sd1342s333dsdsd38` (string, , nullable)
```
Comment (Yarn-e) [inline src/06-products/products.apib:44]:
```suggestion
            + code: `1523999er2323712sd1342s333dsdsd38` (string, nullable)
```

File: apiary.apib [hunk 3]
```diff
@@ -2719,6 +2719,25 @@ Get a list of products.
...
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+
+ + Response 201 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+            + name: `cookies` (string, nullable)
+            + description: `dark chocolate` (string, nullable)
+            + code: `1523999er2323712sd1342s333dsdsd38` (string, , nullable)
```
Comment (Yarn-e) [inline apiary.apib:2738]:
```suggestion
            + code: `1523999er2323712sd1342s333dsdsd38` (string, nullable)
```

**Comment by steftrenson** [comment]
Will be documented in the new release, closing this PR

**Pull request closed without merging**